lib/tic_tac_toe.rb

```diff
@@ -1,26 +1,39 @@
-# frozen_string_literal: true
-
 require_relative './cli'
 require_relative './board'
+require_relative './player'
 
-# TicTacToe is the highest level class and handles the gameplay
 class TicTacToe
   def initialize(user_interface)
     @user_interface = user_interface
     @board = Board.new
+    @player_one = Player.new(@user_interface, 'X')
```

Comment:
I think creating new objects should not happen in initializer

Reply:
Do you think you could let me know why this is a bad practice?

Would it be bad for `initialize` to call another method such as `initialize_player`?